docs/windows.md

```diff
@@ -206,65 +162,60 @@ sed "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"${KUBE_APISERV
 kubectl apply -f -
 ```
 
-Since Antrea 1.13, you can deploy both antrea-agent and antrea-ovs Windows DaemonSets
-with containerd runtime by applying file `antrea-windows-containerd-with-ovs.yml`. The
-following commands download the manifest, set kubeAPIServerOverride, and deploy
-the antrea-agent and antrea-ovs Windows DaemonSets with containerd runtime:
-
-```bash
-KUBE_APISERVER=$(kubectl config view -o jsonpath='{.clusters[0].cluster.server}') && \
-curl -sL https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows-containerd-with-ovs.yml | \
-sed "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"${KUBE_APISERVER}\"|g" | \
-kubectl apply -f -
-```
+When using `antrea-windows-containerd.yml`, you will need to install OVS
+userspace daemons as services when you prepare your Windows worker Nodes, in the
+next section.
 
 #### Join Windows worker Nodes
 
 ##### 1. (Optional) Install OVS (provided by Antrea or your own)
 
-Antrea supports running OVS on Windows as native services or inside a
-host-process container. If you have an OVS package with a signed kernel
-driver and want to run OVS inside container, you can skip this step.
+Depending on which method you are using to install Antrea on Windows, and
+depending on whether you are using your own [signed](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/driver-signing)
+OVS kernel driver or you want to use the test-signed driver provided by Antrea,
+you will need to invoke the `Install-OVS.ps1` script differently (or not at all).
 
-Antrea provides a pre-built OVS package which contains test-signed OVS kernel
-driver. If you don't have a self-signed OVS package and just want to try the
-Antrea on Windows, this package can be used for testing. We also provide a helper
-script `Install-OVS.ps1` to install the OVS driver and register userspace binaries
-as services.
+| Containerized OVS daemons? | Test-signed OVS driver? | Run this command |
+| -------------------------- | ----------------------- | ---------------- |
+| Yes                        | Yes                     | `.\Install-OVS.ps1 -InstallUserspace $false` |
+| Yes                        | No                      | N/A |
```

Comment:
In the current `Install-OVS` script, even if we skip 'InstallUserspace', the OVS driver can still be installed on the host with GUI support. I recall that for non-signed driver installation, a popup window appears when installing the driver, but it should still be installable with the GUI. I'm not sure if it's better to add this information instead of marking it as N/A in this table?

Reply:
I'm confused by your comment. This is for the case where users don't want to use the test-signed drivers provided by Antrea. Are you saying that they should run:
```
.\Install-OVS.ps1 -InstallUserspace $false -ImportCertificate $false -Local -LocalFile <PathToOVSPackage>
```

For a signed driver, the DaemonSet should ideally take care of loading the driver: https://github.com/antrea-io/antrea/blob/main/build/yamls/antrea-windows-containerd-with-ovs.yml#L48-L55
I assume that in this case the driver needs to be placed in a specific location, which unfortunately we do not document AFAIK. Maybe in a future PR?

But my understanding is that for this case (Containerized OVS daemons? Yes - Test-signed OVS driver? No), the intent was that users don't need to run `Install-OVS.ps1`.

Reply:
> For a signed driver, the DaemonSet should ideally take care of loading the driver: https://github.com/antrea-io/antrea/blob/main/build/yamls/antrea-windows-containerd-with-ovs.yml#L48-L55 I assume that in this case the driver needs to be placed in a specific location, which unfortunately we do not document AFAIK. Maybe in a future PR?

Got your point, I am okay to document it in a future PR. Also, since the popup windows issue has been fixed, maybe in this future PR we can also check if it's possible to install test-signed driver during ovs container deployment. If it work we can skip running Install-OVS for test-signed OVS driver as well.

Reply:
>  in this future PR we can also check if it's possible to install test-signed driver during ovs container deployment. If it work we can skip running Install-OVS for test-signed OVS driver as well.

that's a good point, I'll open an issue to track this